uber/templates/preregistration/preregbase.html

```diff
@@ -1,35 +1,135 @@
 {% extends "base.html" %}
-{% block backlink %}
-
-
-{% endblock %}
+{% block backlink %}{% endblock %}
 {% block header %}
-<link href="../static/theme/prereg.css" rel="stylesheet">
-<script type="text/javascript">
-    var checkCap = function () {
-        $.getJSON('../preregistration/check_prereg', function(check) {
-            if (check.force_refresh) {
-                location.reload();  // Reload the page to prevent registering after reg is closed
-            }
-        });
-    };
-    setInterval(checkCap, 1000 * 1 * 90);
+    <link href="../static/theme/prereg.css" rel="stylesheet">
+    <script type="text/javascript">
+        var PREREG_CHECK_INTERVAL = 15 * 60 * 1000;
+        var checkCap = function () {
+            $.getJSON('../preregistration/check_prereg', function (check) {
+                if (check.force_refresh) {
+                    location.reload();  // Reload the page to prevent registering after reg is closed
+                } else {
+                    setTimeout(checkCap, PREREG_CHECK_INTERVAL);
+                }
+            });
+        };
+        setTimeout(checkCap, PREREG_CHECK_INTERVAL);
 
-{% if c.COLLECT_FULL_ADDRESS %}
-$(document).ready(function() {
-    $("#international").hide()
-    setInternational()
-});
+        {% if c.COLLECT_FULL_ADDRESS %}
+            var setInternational = function () {
+                countryName = $('input[name="country"]').val();
+                if(countryName == 'USA' || countryName == 'US' || countryName == 'United States') {
+                    $('input[name="international"]').prop('checked', false);
+                } else {
+                    $('input[name="international"]').prop('checked', true);
+                }
+            }
+            $(function() {
+                $('#international').hide()
+                setInternational()
+            });
+        {% endif %}
 
-function setInternational() {
-    countryName = $("input[name='country']").val();
```

Comment:
Below is the new code that was moved here from ``form.html``.  There are now two global Javascript variables:
- ``REG_TYPES`` controls the first row and currently just determines whether it's a single or a group registration
- ``BADGE_TYPES`` controls the donation level

Both of the above can be overridden in plugins as before.  The main difference is that these two variables each contains several fields:
- ``row`` is the id which is where the options will get appended
- ``selector`` is the class name of the items in that row
- ``options`` is the array of objects which define each option

The main difference between before a